### PR TITLE
Adds feature on shows landing page to collections

### PR DESCRIPTION
**Pull request by kellgauthier**
## Task 
Adds featured on shows landing page to collections within contentful. 
[Rally](https://rally1.rallydev.com/#/66096747656ud/custom/24109743995?detail=%2Fuserstory%2F483183596816)

### CODE REVIEW

File: migrations/20210201160733_add_shows_to_collection.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    with_space do |space|
+      content_type = space.content_types.find('collection')
+
+      field = content_type.fields.detect { |f| f.id == 'featured_on_shows_landing_page' }
+      field.omitted = true
+      field.disabled = true
+
+      content_type.save
+      content_type.activate
+      content_type.fields.destroy('featured_on_shows_landing_page')
```
Comment (mjvezzani) [inline migrations/20210201160733_add_shows_to_collection.rb:21]: Apologies for my ignorance and unfamiliarity with contentful migrations, but does this line need to go above lines 19 & 20? Seems you would want to destroy a field, and then save and activate the 'collection' content type.

**Review by mjvezzani — COMMENTED** [review]
Please see comment about line 21 in this file.

File: migrations/20210201160733_add_shows_to_collection.rb (hunk 1 above)
Comment (silasbaur) [reply migrations/20210201160733_add_shows_to_collection.rb:21]: That is correct. @kellgauthier Can you fix this

Comment (silasbaur) [reply migrations/20210201160733_add_shows_to_collection.rb:21]: @kellgauthier look at other migrations as an example. Once you have this done correctly you can test the migration in INT. Let me know if you want help doing that

Comment (kellgauthier) [reply migrations/20210201160733_add_shows_to_collection.rb]: Resolved. Thank you both!

**Review by mjvezzani — APPROVED** [review]

**Review by silasbaur — APPROVED** [review]

**Pull request merged**